### ISSUE TITLE
Error generating...

### ISSUE THREAD

**Issue by glxiong** [pydanny/cookiecutter-django#1454]
version [0.1.0]: 
timezone [UTC]: 
use_whitenoise [y]: n
use_celery [n]: 
use_mailhog [n]: 
use_sentry_for_error_reporting [y]: n
use_opbeat [n]: 
use_pycharm [n]: 
windows [n]: 
use_docker [n]: 
use_heroku [n]: 
use_compressor [n]: 
Select postgresql_version:
1 - 10
2 - 9.6
3 - 9.5
4 - 9.4
5 - 9.3
6 - 9.2
Choose from 1, 2, 3, 4, 5, 6 [1]: 2
Select js_task_runner:
1 - Gulp
2 - Grunt
3 - None
Choose from 1, 2, 3 [1]: 1
custom_bootstrap_compilation [n]: 
Select open_source_license:
1 - MIT
2 - BSD
3 - GPLv3
4 - Apache Software License 2.0
5 - Not open source
Choose from 1, 2, 3, 4, 5 [1]: 5
Traceback (most recent call last):
  File "/var/folders/hy/cd1snw510vz8x__np0xg34_h3sjm76/T/tmpf_jkcggi.py", line 214, in <module>
    remove_docker_files()
  File "/var/folders/hy/cd1snw510vz8x__np0xg34_h3sjm76/T/tmpf_jkcggi.py", line 134, in remove_docker_files
    PROJECT_DIRECTORY, filename
FileNotFoundError: [Errno 2] No such file or directory: '/Users/xxx/Data/ABC/code/hradvisor/dev.yml'
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)

**Comment by gtracker64** [comment]
I have the same problem.

* **I'm submitting a ... **
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

* **Do you want to request a *feature* or report a *bug*?**

bug

* **What is the current behavior?**

```
Traceback (most recent call last):
  File "/var/folders/bs/s36_92l53nlfcylh7srnd8200000gn/T/tmpykpdgmmk.py", line 214, in <module>
    remove_docker_files()
  File "/var/folders/bs/s36_92l53nlfcylh7srnd8200000gn/T/tmpykpdgmmk.py", line 134, in remove_docker_files
    PROJECT_DIRECTORY, filename
FileNotFoundError: [Errno 2] No such file or directory: '/Users/<USERNAME>/git/project_name/dev.yml'
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** 

```
(virtual_env) macbook:git <USERNAME>$ python --version
Python 3.6.2
(virtual_env) macbook:git <USERNAME>$ cookiecutter --version
Cookiecutter 1.6.0 from /Users/<USERNAME>/.virtualenvs/<project_name>/lib/python3.6/site-packages (Python 3.6)
(virtual_env) macbook:git <USERNAME>$ cookiecutter https://github.com/pydanny/cookiecutter-django
You've downloaded /Users/<USERNAME>/.cookiecutters/cookiecutter-django before. Is it okay to delete and re-download it? [yes]: n
Do you want to re-use the existing version? [yes]: yes
project_name [Project Name]: 
project_slug [project_name]: 
author_name [Daniel Roy Greenfeld]: 
email [you@example.com]: 
description [A short description of the project.]: 
domain_name [example.com]: 
version [0.1.0]: 
timezone [UTC]: 
use_whitenoise [y]: 
use_celery [n]: 
use_mailhog [n]: 
use_sentry_for_error_reporting [y]: 
use_opbeat [n]: 
use_pycharm [n]: 
windows [n]: 
use_docker [n]: 
use_heroku [n]: 
use_compressor [n]: 
Select postgresql_version:
1 - 10
2 - 9.6
3 - 9.5
4 - 9.4
5 - 9.3
6 - 9.2
Choose from 1, 2, 3, 4, 5, 6 [1]: 
Select js_task_runner:
1 - Gulp
2 - Grunt
3 - None
Choose from 1, 2, 3 [1]: 
custom_bootstrap_compilation [n]: 
Select open_source_license:
1 - MIT
2 - BSD
3 - GPLv3
4 - Apache Software License 2.0
5 - Not open source
Choose from 1, 2, 3, 4, 5 [1]: 
Traceback (most recent call last):
  File "/var/folders/bs/s36_92l53nlfcylh7srnd8200000gn/T/tmpykpdgmmk.py", line 214, in <module>
    remove_docker_files()
  File "/var/folders/bs/s36_92l53nlfcylh7srnd8200000gn/T/tmpykpdgmmk.py", line 134, in remove_docker_files
    PROJECT_DIRECTORY, filename
FileNotFoundError: [Errno 2] No such file or directory: '/Users/<USERNAME>/git/project_name/dev.yml'
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)

```


* **What is the expected behavior?**

The project_folder is created on the current directory.

* **What is the motivation / use case for changing the behavior?**

I can created the project_folder when I choose 'y' in use_docker. 
> use_docker [n]: y

* **Please tell us about your environment:**

  * OSX High Sierra 10.13.2
  * Python 3.6.2
  * Cookiecutter 1.6.0

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

remove_docker_files() try to remove some files, but it maybe couldn't find these files in {{cookiecutter.project_slug}} directory.

**Comment by pydanny** [comment]
I believe I fixed this in e69ceebfc584373bfec446033dd1de2fa48560fe. Please test!

**Comment by gtracker64** [comment]
It has been fixed !!
